# Request
float32 degree_turn      # Angle to turn
---
#Response
sensor_msgs/Image camera_image  # image